src/dataLabel/dataLabelUtils.ts

```diff
@@ -235,7 +237,12 @@ export function drawDefaultLabelsForDataPointChart(data: any[], context: Selecti
     }
     else {
         labels
-            .attr({ x: (d: LabelEnabledDataPoint) => d.labelX, y: (d: LabelEnabledDataPoint) => d.labelY })
+            .attr(
+                "x", (d: LabelEnabledDataPoint) => d.labelX
```

Comment:
Since it's less than 120 symbol we should write this function call into a single line